stage2/00-sys-tweaks/00-packages-nr

```diff
@@ -1 +1,2 @@
+nfs-common
```

Comment:
Why was this removed to `00-packages-nr` and how does `00-packages-nr` differ from `00-packages`?

Reply:
Nr doesn't install recommended packages. Read the commit description for why it was moved.

Reply:
Ahh, sorry, missed that commit message.